src/components/GridKeyboardNavigationContext/__tests__/GridKeyboardNavigationContext.jest.js

```diff
@@ -73,31 +70,6 @@ describe("GridKeyboardNavigationContext", () => {
       expect(fakeUpperContext.onOutboundNavigation).toHaveBeenCalledWith(wrapperRef, keyboardDirection);
     });
 
-    it("should focus the element in the direction of focus, when the wrapper element is focused with direction", () => {
```

Comment:
I think these tests should be updated to the new behavior (calling `focus` instead of `focusElementWithDirection`), and not to be removed...

Reply:
[Pushed an update to the the tests + a small fix to the implementation](https://github.com/mondaycom/monday-ui-react-core/pull/586/commits/86030bc3740bafe483fb98149d57b1d8941addfb)